explorer/package.json

```diff
@@ -9,6 +9,7 @@
     "@testing-library/jest-dom": "^5.11.4",
     "@testing-library/react": "^11.0.4",
     "@testing-library/user-event": "^12.1.7",
+    "@theronin/solarweave": "^1.4.0",
```

Comment:
Maybe we publish this under `@solana`? I'm not sure what the handoff / maintenance going forward looks like :)

Reply:
Yea would be great! Feel free to make a new repo and then add me as a Collaborator and I will move the codebase over there.